auth.go

```diff
@@ -444,6 +452,39 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 	return sessionkey, token, nil
 }
 
+func auditUserAction(identity, password, actionDescription string) error {
+	auditServiceUrl := "http://localhost:2016/auth-logaction"
```

Comment:
I feel like auth.go is becoming quite big, perhaps we should create another file audit.go where this function can live?